content/hi/docs/reference/glossary/app-container.md

```diff
@@ -0,0 +1,20 @@
+---
+title: ऐप कंटेनर (App Container)
+id: app-container
+date: 2019-02-12
+full_link:
+short_description: >
+  एक कंटेनर एक कार्यभार का हिस्सा चलाने के लिए प्रयोग किया जाता है। प्रारंभिक कंटेनर की तुलना में।
+
+aka:
+tags:
+- workload
+---
+एप्लिकेशन {{<glossary_tooltip text="पात्र" term_id="container" >}} (या ऐप कंटेनर) {{<glossary_tooltip text="पॉड" term_id="pod" >}} में, {{<glossary_tooltip text ="इनिट कंटेनर" term_id="init-container" >}} स्टार्टअप पूरा होने के बाद शुरू होता है।
```

Comment:
Why have we used पात्र as the text here when we're using container in its machine translated form everywhere else?